modules/sdk-coin-arbeth/test/unit/arbeth.ts

```diff
@@ -585,6 +585,38 @@ describe('Arbitrum', function () {
       recipient.amount.should.equal('9999999999999999928');
     });
 
+    it('should generate an unsigned sweep', async function () {
```

Comment:
Should throw error should be the test description?

Reply:
my bad, updated